metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImplTest.java

```diff
@@ -60,9 +57,9 @@
 public class DereferencerImplTest {
 
   private static final String[] DEREFERENCE_EXTRACT_RESULT_INVALID = {
-      "http://invalid-example.host/about",
-      "http://invalid-example.host/concept",
-      "http://invalid-example.host/place"};
```

Comment:
Perhaps a good check would be a space character in the URL (either in the path or in the query): this happens a lot and should trigger a warning.

Reply:
done. added URI checking as well.